### PR TITLE
[Docs engine] Parrot stripe's api doc link labeling

### PR DESCRIPTION
**Pull request by kodster28**
- Add new rendering logic to include an SVG image next to API doc links (similar to [Stripe's docs](https://stripe.com/docs/payments/checkout/managing-limited-inventory#manual-expiration))
- Also, clean up references to developers.cloudflare.com/api/
Screenshot 2023-07-12 at 4 00 11 PM

### CODE REVIEW

**Comment by pedrosousa** [comment]
We probably need to prevent the specific API link in the homepage from using the new style, or at least we'll have to keep the SVG on the same line.
image
([page link](https://highlight-api-links.cloudflare-docs-7ou.pages.dev/))

Also, API links in the left sidebar kept their "external link" icon, instead of using the new SVG — not sure if this is something we also wish to change:
image
([page link](https://highlight-api-links.cloudflare-docs-7ou.pages.dev/email-routing/))

**Comment by kodster28** [comment]
> We probably need to prevent the specific API link in the homepage from using the new style, or at least we'll have to keep the SVG on the same line. image ([page link](https://highlight-api-links.cloudflare-docs-7ou.pages.dev/))
> 
> Also, API links in the left sidebar kept their "external link" icon, instead of using the new SVG — not sure if this is something we also wish to change: image ([page link](https://highlight-api-links.cloudflare-docs-7ou.pages.dev/email-routing/))

Nice! I'll fix those, both good catches.

**Review by pedrosousa — APPROVED** [review]

File: content/firewall/api/cf-filters/endpoints.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ layout: list
 
 {{<content-column>}}
 
-To invoke a Cloudflare Filters API operation, append the endpoint to the Cloudflare API base URL:
+To invoke a Cloudflare Filters operation, append the endpoint to the Cloudflare API base URL:
```
Comment (kodster28) [inline content/firewall/api/cf-filters/endpoints.md]:
```suggestion
To invoke a Cloudflare Filters API operation, append the endpoint to the Cloudflare API base URL:
```

File: content/firewall/api/cf-firewall-rules/endpoints.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ layout: list
 
 {{<content-column>}}
 
-To invoke a Cloudflare Firewall Rules API operation, append the endpoint to the Cloudflare API base URL:
+To invoke a Cloudflare Firewall Rules operation, append the endpoint to the Cloudflare API base URL:
```
Comment (kodster28) [inline content/firewall/api/cf-firewall-rules/endpoints.md]:
```suggestion
To invoke a Cloudflare Firewall Rules API operation, append the endpoint to the Cloudflare API base URL:
```

File: content/fundamentals/global-configurations/lists/lists-api/endpoints.md [hunk 3]
```diff
@@ -114,7 +114,7 @@ The Lists API supports the operations outlined below. Visit the associated links
 
 Nearly all the operations for managing items in a list are asynchronous. When you add or delete a large amount of items to or from a list, there may be a delay before the bulk operation is complete.
 
-Asynchronous list operations return an `operation_id`, which you can use to monitor the status of an API operation. To monitor the status of an asynchronous operation, use the [Get bulk operation status](/api/operations/lists-get-bulk-operation-status) endpoint and specify the ID of the operation you want to monitor.
+Asynchronous list operations return an `operation_id`, which you can use to monitor the status of an operation. To monitor the status of an asynchronous operation, use the [Get bulk operation status](/api/operations/lists-get-bulk-operation-status) endpoint and specify the ID of the operation you want to monitor.
```
Comment (kodster28) [inline content/fundamentals/global-configurations/lists/lists-api/endpoints.md]:
```suggestion
Asynchronous list operations return an `operation_id`, which you can use to monitor the status of an API operation. To monitor the status of an asynchronous operation, use the [Get bulk operation status](/api/operations/lists-get-bulk-operation-status) endpoint and specify the ID of the operation you want to monitor.
```

File: content/page-shield/reference/page-shield-api.md [hunk 4]
```diff
@@ -26,7 +26,7 @@ The `{zone_id}` argument is the zone ID (a hexadecimal string). You can find thi
 
 The `{script_id}` argument is the script ID (a hexadecimal string). This value is included in the response of the [List Page Shield scripts](/api/operations/page-shield-list-page-shield-scripts) operation for every detected script.
 
-The `{connection_id}` argument is the connection ID (a hexadecimal string). This value is included in the response of the List Page Shield connections API operation for every detected connection.
+The `{connection_id}` argument is the connection ID (a hexadecimal string). This value is included in the response of the List Page Shield connections operation for every detected connection.
```
Comment (kodster28) [inline content/page-shield/reference/page-shield-api.md]:
```suggestion
The `{connection_id}` argument is the connection ID (a hexadecimal string). This value is included in the response of the List Page Shield connections API operation for every detected connection.
```

File: content/rules/reference/esc-deprecation.md [hunk 5]
```diff
@@ -104,7 +104,7 @@ Re-enable ESC for an account            | DELETE    | `/accounts/<ACCOUNT_ID>/di
 
 {{</table-wrap>}}
 
-To invoke an API operation, append the operation endpoint to the Cloudflare API base URL:
+To invoke an operation, append the operation endpoint to the Cloudflare API base URL:
```
Comment (kodster28) [inline content/rules/reference/esc-deprecation.md]:
```suggestion
To invoke an API operation, append the operation endpoint to the Cloudflare API base URL:
```

File: content/ruleset-engine/custom-rulesets/deploy-custom-ruleset.md [hunk 6]
```diff
@@ -17,7 +17,7 @@ Before you begin:
 To deploy a custom ruleset, add a rule that executes the custom ruleset. Define the rule scope in the rule expression.
 
 {{<Aside type="warning">}}
-Regarding the expression of the rule deploying the ruleset, you must use parentheses to enclose any custom conditions and end your expression with `and cf.zone.plan eq "ENT"` or else the API operation will fail.
+Regarding the expression of the rule deploying the ruleset, you must use parentheses to enclose any custom conditions and end your expression with `and cf.zone.plan eq "ENT"` or else the operation will fail.
```
Comment (kodster28) [inline content/ruleset-engine/custom-rulesets/deploy-custom-ruleset.md]:
```suggestion
Regarding the expression of the rule deploying the ruleset, you must use parentheses to enclose any custom conditions and end your expression with `and cf.zone.plan eq "ENT"` or else the API operation will fail.
```

File: content/ruleset-engine/rulesets-api/endpoints.md [hunk 7]
```diff
@@ -20,7 +20,7 @@ You can use the following endpoint:
 PUT /zones/{zone_id}/rulesets/phases/{phase_name}/entrypoint
 ```
 
-To invoke a Rulesets API operation, append the endpoint to the Cloudflare API base URL:
+To invoke a Rulesets operation, append the endpoint to the Cloudflare API base URL:
```
Comment (kodster28) [inline content/ruleset-engine/rulesets-api/endpoints.md]:
```suggestion
To invoke a Rulesets API operation, append the endpoint to the Cloudflare API base URL:
```

File: content/waf/custom-rules/skip/api-examples.md [hunk 8]
```diff
@@ -20,13 +20,13 @@ This page contains examples of different skip rule scenarios for custom rules. T
 
 * The `{ruleset_id}` value is the ID of the [entry point ruleset](/ruleset-engine/about/phases/#phase-entry-point-ruleset) of the `http_request_firewall_custom` phase. For details on obtaining this ruleset ID, refer to [List and view rulesets](/ruleset-engine/rulesets-api/view/). The API examples in this page add a skip rule to an existing ruleset using the [Create a zone ruleset rule](/api/operations/createZoneRulesetRule) operation.
 
-    However, the entry point ruleset may not exist yet. In this case, invoke the [Create a zone ruleset](/api/operations/createZoneRuleset) API operation to create the entry point ruleset with a skip rule. Refer to [Create ruleset](/ruleset-engine/rulesets-api/create/#example---create-a-zone-level-phase-entry-point-ruleset) for an example.
+    However, the entry point ruleset may not exist yet. In this case, invoke the [Create a zone ruleset](/api/operations/createZoneRuleset) operationcreate the entry point ruleset with a skip rule. Refer to [Create ruleset](/ruleset-engine/rulesets-api/create/#example---create-a-zone-level-phase-entry-point-ruleset) for an example.
```
Comment (kodster28) [inline content/waf/custom-rules/skip/api-examples.md]:
```suggestion
    However, the entry point ruleset may not exist yet. In this case, invoke the [Create a zone ruleset](/api/operations/createZoneRuleset) operation to create the entry point ruleset with a skip rule. Refer to [Create ruleset](/ruleset-engine/rulesets-api/create/#example---create-a-zone-level-phase-entry-point-ruleset) for an example.
```

File: content/waf/custom-rulesets/create-api.md [hunk 9]
```diff
@@ -21,7 +21,7 @@ For more information, refer to [Work with custom rulesets](/ruleset-engine/custo
 
 {{<Aside type="warning" header="Important">}}
 
-* Deployed custom rulesets will only apply to incoming traffic of Enterprise domains. Regarding the expression of the rule deploying the custom ruleset, you must use parentheses to enclose any custom conditions and end your expression with `and cf.zone.plan eq "ENT"` or else the API operation will fail.
+* Deployed custom rulesets will only apply to incoming traffic of Enterprise domains. Regarding the expression of the rule deploying the custom ruleset, you must use parentheses to enclose any custom conditions and end your expression with `and cf.zone.plan eq "ENT"` or else the operation will fail.
```
Comment (kodster28) [inline content/waf/custom-rulesets/create-api.md]:
```suggestion
* Deployed custom rulesets will only apply to incoming traffic of Enterprise domains. Regarding the expression of the rule deploying the custom ruleset, you must use parentheses to enclose any custom conditions and end your expression with `and cf.zone.plan eq "ENT"` or else the API operation will fail.
```

**Pull request merged**